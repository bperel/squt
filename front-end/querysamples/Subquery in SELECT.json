{
   "Tables" : {
      "a" : {
         "a" : {
            "OUTPUT" : {
               "b" : {
                  "-1" : "b"
               }
            }
         }
      }
   },
   "Subqueries" : {
      "0" : {
         "SubqueryAlias" : "e",
         "Tables" : {
            "c" : {
               "c" : {
                  "OUTPUT" : {
                     "d" : {
                        "-1" : "d"
                     }
                  }
               }
            }
         }
      }
   }
}

