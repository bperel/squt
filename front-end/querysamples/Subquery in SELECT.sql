SELECT a.b, 
	(SELECT c.d FROM c) e 
FROM a
