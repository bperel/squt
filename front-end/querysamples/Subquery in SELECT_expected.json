{
   "Subqueries" : {
      "0" : {
         "SubqueryAlias" : "e",
         "Tables" : {
            "c" : {
               "c" : {
                  "OUTPUT" : {
                     "d" : {
                        "-1" : "d"
                     }
                  }
               }
            }
         },
         "SubqueryType" : "SINGLEROW_SUBS"
      }
   },
   "Tables" : {
      "a" : {
         "a" : {
            "OUTPUT" : {
               "b" : {
                  "-1" : "b"
               }
            }
         }
      }
   }
}
